modelservice/src/main/sql/build-db.sql

```diff
@@ -7,10 +7,11 @@ use aa_model_service;
 
 create table metric
 (
-  id        int unsigned primary key not null auto_increment,
-  m_key     varchar(100) unique not null,
-  hash      varchar(100) unique not null,
-  tags      json
+  id           int unsigned primary key not null auto_increment,
+  m_key        varchar(100) unique not null,
+  hash         varchar(100) unique not null,
+  description  varchar(100) unique not null,
```

Comment:
description should probably be longer than 100. I could imagine somebody wanting to describe the data source, or putting a link to a doppler URL, etc. can we do 255 here?